tests/profile/profile-page-uploads-facets.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { test } from '../fixtures';
 
-test(`Profile Page - Uploads: facets appear`, async ({
+test.fixme(`Profile Page - Uploads: facets appear`, async ({
```

Comment:
Q: does this run the test?

Reply:
tests with fixme annotations will be skipped